### ISSUE TITLE
All browsers are crashing

### ISSUE THREAD

**Issue by manab52425** [PixelExperience/android-issues#136]
## Build date

```
PixelExperience_Plus_lavender-11.0-20210526-1452-OFFICIAL
```

## Expected Behavior

```
All browsers are running smoothly
```

## Current Behavior

```
Browsers are crashing everytime when I use it
```

## Possible Solution

```
An update should fix it
```

## Steps to Reproduce

```
1. After clean reflashing the latest build I'm getting this problem only on my device.
2.
3.
4.
```

/device lavender
/version eleven_plus
[logcat.txt](https://github.com/PixelExperience/android-issues/files/6624279/logcat.txt)
![Screenshot_20210604-132733_Pixel_Launcher](https://user-images.githubusercontent.com/84611056/120853757-c32b6000-c599-11eb-9a6e-b5dee4b20218.png)
![Screenshot_20210604-132557_Pixel_Launcher](https://user-images.githubusercontent.com/84611056/120853782-cd4d5e80-c599-11eb-80c9-e115b6540686.png)

**Comment by manab52425** [comment]
Please fix this issue on June update

**Comment by faham1997** [comment]
Is fixed on  June Security patch . Wait for update!

**Comment by faham1997** [comment]
Has been fixed on PixelExperience_Plus_lavender-11.0-20210618-2115-OFFICIAL.zip
